server/tableNamespaceClassifier.go

```diff
@@ -0,0 +1,46 @@
+package server
```

Comment:
use `table_namespace_classifier.go` , we don't support CamelCase for the file name.

Reply:
Also please add license header.